database/migrations/2024_10_18_111330_add_total_reserved_graduate_attestations_field_to_fed_caps_table.php

```diff
@@ -12,7 +12,7 @@
     public function up(): void
     {
         Schema::table('fed_caps', function (Blueprint $table) {
-            $table->string('total_reserved_graduate_attestations')->nullable();
+            $table->integer('total_reserved_graduate_attestations')->nullable();
```

Comment:
instead of nullable() switch it to default(0)

Reply:
That way you always compare against numbers and you don't have to deal with different types